src/cdk-experimental/scrolling/virtual-scroll.md

```diff
@@ -0,0 +1,140 @@
+**Warning: this component is still experimental. It may have bugs and the API may change at any
+time**
+
+`<cdk-virtual-scroll-viewport>` is used to display a scrolling list containing a large number of
+items that would have a negative effect on performance if all rendered at once. Instead the virtual
+scroll viewport renders each item as needed when the user scrolls it into view. The
+`<cdk-virtual-scroll-viewport>` works together with the `*cdkVirtualFor` directive which is used to
+render items inside of the viewport. It also requires that a `VirtualScrollStrategy` be provided.
+The easiest way to provide a strategy is with one of the directives `itemSize` or `autosize`.
+
+For some example usages, 
+[see the demo app](https://github.com/angular/material2/tree/master/src/demo-app/virtual-scroll).
+
+### Creating items in the viewport
+`*cdkVirtualFor` should be used instead of `*ngFor` to create items in the
+`<cdk-virtual-scroll-viewport>`. `*cdkVirtualFor` supports most of the same features as `*ngFor`.
```

Comment:
Should it mention which features it doesn't support?

Reply:
I wrote that because I'm naturally hedgey, but I think it supports all of the same features as `*ngFor`, updated to say that.